sphinx_gallery/_static/gallery.css

```diff
@@ -115,12 +120,14 @@ div.sphx-glr-download a {
   border: 1px solid #c2c22d;
   color: #000;
   display: inline-block;
+  /* Not valid in old browser, hence we keep the line above to override */
+  display: table-caption;
   font-weight: bold;
-  max-width: 45ex;
   padding: 1ex;
   text-align: center;
 }
 
+
 div.sphx-glr-download code.download {
   display: inline-block;
 }
```

Comment:
To solve the conflict for scikit-learn with the boostrap.min.css we need to add an extra `white-space: normal;` in this class. Or shall that be kept only in scikit-learn css files?